package.json

```diff
@@ -42,6 +42,28 @@
     ],
     "main": "./out/extension",
     "contributes": {
+        "languages": [
+            {
+                "id": "yaml",
+                "aliases": [
+                    "YAML",
+                    "yaml"
+                ],
+                "extensions": [
+                    ".yml",
+                    ".eyaml",
+                    ".eyml",
+                    ".yaml"
+                ]
+            }
+        ],
+        "grammars": [
```

Comment:
what is the purpose of grammar?

Reply:
The grammar is used for syntax validation. Here, by providing the path to `yaml.tmLanguage.json`, we are specifying the grammar to be used for syntax validation.